client/src/components/KeyConceptList/KeyConceptList.tsx

```diff
@@ -6,7 +6,7 @@ import "./KeyConceptList.scss";
 
 interface KeyConceptListProps {
   question_answer_pairs: [React.ReactNode, React.ReactNode][];
-  compact: boolean;
+  compact?: boolean;
```

Comment:
Hmm, looking at this code I'm not sure of the real use of having `compact` but that can be dealt with later when working on the PinnedFAQ visual bug you mentioned